### PR TITLE
r/s3_bucket: read-only `request_payer` argument

### PR DESCRIPTION
**Pull request by anGie44**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #4418
Relates #20433

Output from acceptance testing:

```
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_empty (16.31s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_multipleTags (173.68s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefix (169.69s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefixAndTags (173.14s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_remove (170.47s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_singleTag (171.69s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_default (103.50s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_empty (15.29s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_full (103.74s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_basic (102.27s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_removed (157.94s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_updateBasic (243.40s)

--- PASS: TestAccS3BucketDataSource_basic (91.85s)
--- PASS: TestAccS3BucketDataSource_website (90.14s)

--- PASS: TestAccS3BucketIntelligentTieringConfiguration_Filter (380.33s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_basic (102.69s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_disappears (90.45s)

--- PASS: TestAccS3BucketInventory_basic (101.98s)
--- PASS: TestAccS3BucketInventory_encryptWithSSEKMS (101.66s)
--- PASS: TestAccS3BucketInventory_encryptWithSSES3 (102.98s)

--- PASS: TestAccS3BucketMetric_basic (101.70s)
--- PASS: TestAccS3BucketMetric_withEmptyFilter (15.89s)
--- PASS: TestAccS3BucketMetric_withFilterMultipleTags (168.83s)
--- PASS: TestAccS3BucketMetric_withFilterPrefix (168.35s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndMultipleTags (169.35s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndSingleTag (169.40s)
--- PASS: TestAccS3BucketMetric_withFilterSingleTag (169.66s)

--- PASS: TestAccS3BucketNotification_LambdaFunctionLambdaFunctionARN_alias (118.09s)
--- PASS: TestAccS3BucketNotification_Topic_multiple (102.58s)
--- PASS: TestAccS3BucketNotification_lambdaFunction (126.87s)
--- PASS: TestAccS3BucketNotification_queue (130.25s)
--- PASS: TestAccS3BucketNotification_topic (100.74s)
--- PASS: TestAccS3BucketNotification_update (187.21s)

--- PASS: TestAccS3BucketObjectDataSource_allParams (89.32s)
--- PASS: TestAccS3BucketObjectDataSource_basic (87.33s)
--- PASS: TestAccS3BucketObjectDataSource_basicViaAccessPoint (90.27s)
--- PASS: TestAccS3BucketObjectDataSource_bucketKeyEnabled (89.18s)
--- PASS: TestAccS3BucketObjectDataSource_kmsEncrypted (90.59s)
--- PASS: TestAccS3BucketObjectDataSource_leadingSlash (155.96s)
--- PASS: TestAccS3BucketObjectDataSource_multipleSlashes (161.08s)
--- PASS: TestAccS3BucketObjectDataSource_objectLockLegalHoldOff (92.20s)
--- PASS: TestAccS3BucketObjectDataSource_objectLockLegalHoldOn (87.86s)
--- PASS: TestAccS3BucketObjectDataSource_readableBody (90.19s)
--- PASS: TestAccS3BucketObjectDataSource_singleSlashAsKey (87.79s)

--- PASS: TestAccS3BucketObject_acl (251.17s)
--- PASS: TestAccS3BucketObject_bucketBucketKeyEnabled (93.17s)
--- PASS: TestAccS3BucketObject_content (99.40s)
--- PASS: TestAccS3BucketObject_contentBase64 (89.08s)
--- PASS: TestAccS3BucketObject_defaultBucketSSE (94.21s)
--- PASS: TestAccS3BucketObject_empty (103.26s)
--- PASS: TestAccS3BucketObject_etagEncryption (98.60s)
--- PASS: TestAccS3BucketObject_ignoreTags (169.46s)
--- PASS: TestAccS3BucketObject_kms (101.22s)
--- PASS: TestAccS3BucketObject_metadata (249.90s)
--- PASS: TestAccS3BucketObject_noNameNoKey (24.45s)
--- PASS: TestAccS3BucketObject_nonVersioned (100.74s)
--- PASS: TestAccS3BucketObject_objectBucketKeyEnabled (91.00s)
--- PASS: TestAccS3BucketObject_objectLockLegalHoldStartWithNone (241.45s)
--- PASS: TestAccS3BucketObject_objectLockLegalHoldStartWithOn (166.03s)
--- PASS: TestAccS3BucketObject_objectLockRetentionStartWithNone (240.41s)
--- PASS: TestAccS3BucketObject_objectLockRetentionStartWithSet (312.30s)
--- PASS: TestAccS3BucketObject_source (102.65s)
--- PASS: TestAccS3BucketObject_sourceHashTrigger (173.25s)
--- PASS: TestAccS3BucketObject_sse (102.00s)
--- PASS: TestAccS3BucketObject_storageClass (394.53s)
--- PASS: TestAccS3BucketObject_tags (328.94s)
--- PASS: TestAccS3BucketObject_tagsLeadingMultipleSlashes (309.65s)
--- PASS: TestAccS3BucketObject_tagsLeadingSingleSlash (323.67s)
--- PASS: TestAccS3BucketObject_tagsMultipleSlashes (309.83s)
--- PASS: TestAccS3BucketObject_updateSameFile (163.03s)
--- PASS: TestAccS3BucketObject_updates (175.36s)
--- PASS: TestAccS3BucketObject_updatesWithVersioning (174.10s)
--- PASS: TestAccS3BucketObject_updatesWithVersioningViaAccessPoint (163.30s)
--- PASS: TestAccS3BucketObject_withContentCharacteristics (86.60s)

--- PASS: TestAccS3BucketObjectsDataSource_all (173.41s)
--- PASS: TestAccS3BucketObjectsDataSource_basic (173.04s)
--- PASS: TestAccS3BucketObjectsDataSource_basicViaAccessPoint (171.15s)
--- PASS: TestAccS3BucketObjectsDataSource_encoded (170.85s)
--- PASS: TestAccS3BucketObjectsDataSource_fetchOwner (171.44s)
--- PASS: TestAccS3BucketObjectsDataSource_maxKeys (167.83s)
--- PASS: TestAccS3BucketObjectsDataSource_prefixes (170.08s)
--- PASS: TestAccS3BucketObjectsDataSource_startAfter (168.14s)

--- PASS: TestAccS3BucketOwnershipControls_Disappears_bucket (79.33s)
--- PASS: TestAccS3BucketOwnershipControls_Rule_objectOwnership (183.71s)
--- PASS: TestAccS3BucketOwnershipControls_basic (103.02s)
--- PASS: TestAccS3BucketOwnershipControls_disappears (91.51s)

--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_jsonEncode (372.66s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDoc (271.44s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDocNotPrincipal (294.43s)

--- PASS: TestAccS3BucketPolicy_basic (102.35s)
--- PASS: TestAccS3BucketPolicy_policyUpdate (187.48s)

--- PASS: TestAccS3BucketPublicAccessBlock_Disappears_bucket (77.59s)
--- PASS: TestAccS3BucketPublicAccessBlock_basic (105.39s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicACLs (250.46s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicPolicy (254.27s)
--- PASS: TestAccS3BucketPublicAccessBlock_disappears (91.09s)
--- PASS: TestAccS3BucketPublicAccessBlock_ignorePublicACLs (253.52s)
--- PASS: TestAccS3BucketPublicAccessBlock_restrictPublicBuckets (249.89s)

--- PASS: TestAccS3BucketReplicationConfiguration_basic (540.64s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAccessControlTranslation (494.84s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAddAccessControlTranslation (486.51s)
--- PASS: TestAccS3BucketReplicationConfiguration_disappears (141.74s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_andOperator (366.45s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_tagFilter (363.38s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsEmptyFilter (407.56s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsNonEmptyFilter (408.67s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicaModifications (400.92s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicationTimeControl (403.87s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2 (395.35s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2DestinationMetrics (363.91s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2SameRegion (233.62s)
--- PASS: TestAccS3BucketReplicationConfiguration_twoDestination (408.12s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutPrefix (320.73s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutStorageClass (398.66s)

--- PASS: TestAccS3BucketVersioning_MFADelete (63.78s)
--- PASS: TestAccS3BucketVersioning_basic (101.01s)
--- PASS: TestAccS3BucketVersioning_disappears (89.84s)
--- PASS: TestAccS3BucketVersioning_update (141.46s)

--- PASS: TestAccS3Bucket_Basic_acceleration (69.50s)
--- PASS: TestAccS3Bucket_Basic_basic (60.40s)
--- PASS: TestAccS3Bucket_Basic_emptyString (58.41s)
--- PASS: TestAccS3Bucket_Basic_forceDestroy (99.82s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithEmptyPrefixes (99.96s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithObjectLockEnabled (100.05s)
--- PASS: TestAccS3Bucket_Basic_generatedName (34.83s)
--- PASS: TestAccS3Bucket_Basic_keyEnabled (69.07s)
--- PASS: TestAccS3Bucket_Basic_namePrefix (39.80s)
--- PASS: TestAccS3Bucket_Basic_shouldFailNotFound (50.01s)

--- PASS: TestAccS3Bucket_Manage_MfaDeleteDisabled (71.04s)
--- PASS: TestAccS3Bucket_Manage_lifecycleBasic (232.34s)
--- PASS: TestAccS3Bucket_Manage_lifecycleExpireMarkerOnly (180.18s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleAbortIncompleteMultipartUploadDaysNoExpiration (104.93s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleExpirationEmptyBlock (86.20s)
--- PASS: TestAccS3Bucket_Manage_objectLock (154.98s)
--- PASS: TestAccS3Bucket_Manage_versioning (151.80s)
--- PASS: TestAccS3Bucket_Manage_versioningAndMfaDeleteDisabled (81.01s)
--- PASS: TestAccS3Bucket_Manage_versioningDisabled (70.02s)
--- PASS: TestAccS3Bucket_Replication_RTC_valid (253.00s)
--- PASS: TestAccS3Bucket_Replication_basic (307.82s)
--- PASS: TestAccS3Bucket_Replication_expectVersioningValidationError (77.47s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsEmptyFilter (171.42s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsNonEmptyFilter (171.99s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAccessControlTranslation (230.33s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAddAccessControlTranslation (220.82s)
--- PASS: TestAccS3Bucket_Replication_schemaV2 (297.91s)
--- PASS: TestAccS3Bucket_Replication_schemaV2SameRegion (118.79s)
--- PASS: TestAccS3Bucket_Replication_twoDestination (174.11s)
--- PASS: TestAccS3Bucket_Replication_withoutPrefix (161.50s)
--- PASS: TestAccS3Bucket_Replication_withoutStorageClass (166.81s)
--- PASS: TestAccS3Bucket_Security_aclToGrant (56.07s)
--- PASS: TestAccS3Bucket_Security_corsDelete (72.98s)
--- PASS: TestAccS3Bucket_Security_corsEmptyOrigin (84.92s)
--- PASS: TestAccS3Bucket_Security_corsUpdate (158.92s)
--- PASS: TestAccS3Bucket_Security_disableDefaultEncryptionWhenDefaultEncryptionIsEnabled (118.38s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenAES256IsUsed (58.93s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenTypical (61.87s)
--- PASS: TestAccS3Bucket_Security_grantToACL (83.03s)
--- PASS: TestAccS3Bucket_Security_logging (94.47s)
--- PASS: TestAccS3Bucket_Security_policy (90.44s)
--- PASS: TestAccS3Bucket_Security_updateACL (57.50s)
--- PASS: TestAccS3Bucket_Security_updateGrant (109.66s)
--- PASS: TestAccS3Bucket_Tags_basic (39.64s)
--- PASS: TestAccS3Bucket_Tags_ignoreTags (59.31s)
--- PASS: TestAccS3Bucket_Tags_withNoSystemTags (136.32s)
--- PASS: TestAccS3Bucket_Tags_withSystemTags (169.10s)
--- PASS: TestAccS3Bucket_Web_redirect (173.83s)
--- PASS: TestAccS3Bucket_Web_routingRules (105.18s)
--- PASS: TestAccS3Bucket_Web_simple (166.50s)

--- PASS: TestBucketName (0.00s)
--- PASS: TestBucketRegionalDomainName (0.00s)
--- PASS: TestWebsiteEndpoint (0.00s)
--- SKIP: TestAccS3BucketReplicationConfiguration_existingObjectReplication (0.00s)
```

### CODE REVIEW

**Comment by anGie44** [comment]
Optionally preceded by #22649

File: internal/service/s3/bucket_test.go [hunk 1]
```diff
@@ -397,42 +397,6 @@ func TestAccS3Bucket_Basic_acceleration(t *testing.T) {
 	})
 }
 
-func TestAccS3Bucket_Basic_requestPayer(t *testing.T) {
```
Comment (anGie44) [inline internal/service/s3/bucket_test.go:400]: Tests have been ported over to #22649

**Review by YakDriver — APPROVED** [review]
Looks great! 🎉 

```console
% make testacc TESTS=TestAccS3Bucket PKG=s3
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/s3/... -v -count 1 -parallel 20 -run='TestAccS3Bucket'  -timeout 180m
--- SKIP: TestAccS3BucketReplicationConfiguration_existingObjectReplication (0.00s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithEmptyPrefixes (67.07s)
--- PASS: TestAccS3Bucket_Basic_forceDestroy (73.59s)
--- PASS: TestAccS3BucketNotification_eventbridge (81.59s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_basic (84.24s)
--- PASS: TestAccS3BucketVersioning_disappears (85.57s)
--- PASS: TestAccS3BucketCorsConfiguration_MultipleRules (86.39s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithObjectLockEnabled (88.09s)
--- PASS: TestAccS3BucketVersioning_basic (96.68s)
--- PASS: TestAccS3BucketVersioning_MFADelete (99.00s)
--- PASS: TestAccS3Bucket_Replication_expectVersioningValidationError (30.31s)
--- PASS: TestAccS3Bucket_Manage_objectLock (142.34s)
--- PASS: TestAccS3BucketReplicationConfiguration_disappears (77.28s)
--- PASS: TestAccS3Bucket_Replication_schemaV2SameRegion (101.71s)
--- PASS: TestAccS3Bucket_Replication_withoutPrefix (105.16s)
--- PASS: TestAccS3BucketPolicy_basic (66.72s)
--- PASS: TestAccS3BucketVersioning_update (200.92s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2SameRegion (205.07s)
--- PASS: TestAccS3BucketOwnershipControls_Disappears_bucket (37.79s)
--- PASS: TestAccS3Bucket_Replication_withoutStorageClass (93.33s)
--- PASS: TestAccS3BucketOwnershipControls_disappears (48.33s)
--- PASS: TestAccS3BucketOwnershipControls_basic (56.50s)
--- PASS: TestAccS3BucketOwnershipControls_Rule_objectOwnership (107.89s)
--- PASS: TestAccS3BucketNotification_Topic_multiple (57.70s)
--- PASS: TestAccS3BucketNotification_topic (56.83s)
--- PASS: TestAccS3Bucket_Replication_RTC_valid (311.64s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAddAccessControlTranslation (139.06s)
--- PASS: TestAccS3Bucket_Replication_twoDestination (72.01s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAccessControlTranslation (143.17s)
--- PASS: TestAccS3BucketPolicy_policyUpdate (72.43s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2DestinationMetrics (346.99s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutStorageClass (347.29s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicationTimeControl (347.53s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2 (348.01s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicaModifications (348.87s)
--- PASS: TestAccS3BucketNotification_update (132.17s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsNonEmptyFilter (69.74s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleExpirationEmptyBlock (48.13s)
--- PASS: TestAccS3BucketReplicationConfiguration_twoDestination (331.20s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleAbortIncompleteMultipartUploadDaysNoExpiration (57.08s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsNonEmptyFilter (336.40s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsEmptyFilter (330.15s)
--- PASS: TestAccS3Bucket_Security_logging (66.94s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsEmptyFilter (84.08s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAddAccessControlTranslation (419.87s)
--- PASS: TestAccS3BucketPublicAccessBlock_restrictPublicBuckets (126.91s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAccessControlTranslation (421.95s)
--- PASS: TestAccS3Bucket_Security_corsEmptyOrigin (67.28s)
--- PASS: TestAccS3Bucket_Security_corsDelete (56.22s)
--- PASS: TestAccS3BucketPublicAccessBlock_ignorePublicACLs (136.96s)
--- PASS: TestAccS3BucketNotification_queue (99.04s)
--- PASS: TestAccS3Bucket_Replication_schemaV2 (362.14s)
--- PASS: TestAccS3Bucket_Basic_shouldFailNotFound (47.34s)
--- PASS: TestAccS3Bucket_Manage_lifecycleExpireMarkerOnly (117.51s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicPolicy (144.17s)
--- PASS: TestAccS3Bucket_Manage_versioningAndMfaDeleteDisabled (68.53s)
--- PASS: TestAccS3Bucket_Manage_MfaDeleteDisabled (67.56s)
--- PASS: TestAccS3Bucket_Manage_versioningDisabled (71.54s)
--- PASS: TestAccS3BucketCorsConfiguration_SingleRule (67.56s)
--- PASS: TestAccS3Bucket_Basic_keyEnabled (84.11s)
--- PASS: TestAccS3BucketCorsConfiguration_disappears (62.19s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_empty (4.52s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenAES256IsUsed (69.39s)
--- PASS: TestAccS3BucketCorsConfiguration_basic (71.06s)
--- PASS: TestAccS3BucketPublicAccessBlock_Disappears_bucket (50.38s)
--- PASS: TestAccS3BucketReplicationConfiguration_basic (425.18s)
--- PASS: TestAccS3Bucket_Security_corsUpdate (131.74s)
--- PASS: TestAccS3Bucket_Manage_lifecycleBasic (179.65s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenTypical (82.52s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_full (71.35s)
--- PASS: TestAccS3BucketPublicAccessBlock_disappears (61.07s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_remove (128.33s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_empty (3.01s)
--- PASS: TestAccS3Bucket_Security_disableDefaultEncryptionWhenDefaultEncryptionIsEnabled (135.22s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_default (72.94s)
--- PASS: TestAccS3Bucket_Manage_versioning (149.68s)
--- PASS: TestAccS3BucketNotification_lambdaFunction (45.12s)
--- PASS: TestAccS3BucketPublicAccessBlock_basic (70.94s)
--- PASS: TestAccS3Bucket_Web_routingRules (134.42s)
--- PASS: TestAccS3BucketNotification_LambdaFunctionLambdaFunctionARN_alias (84.53s)
--- PASS: TestAccS3BucketCorsConfiguration_update (176.19s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefixAndTags (124.50s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefix (125.27s)
--- PASS: TestAccS3Bucket_Tags_basic (77.65s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_multipleTags (126.00s)
--- PASS: TestAccS3Bucket_Basic_emptyString (78.33s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicACLs (184.96s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_removed (105.07s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_singleTag (123.56s)
--- PASS: TestAccS3Bucket_Replication_basic (328.63s)
--- PASS: TestAccS3Bucket_Web_redirect (201.56s)
--- PASS: TestAccS3Bucket_Tags_ignoreTags (47.73s)
--- PASS: TestAccS3Bucket_Web_simple (189.12s)
--- PASS: TestAccS3Bucket_Basic_generatedName (73.18s)
--- PASS: TestAccS3Bucket_Security_aclToGrant (117.82s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_disappears (54.45s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDoc (184.76s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDocNotPrincipal (198.54s)
--- PASS: TestAccS3Bucket_Basic_basic (71.43s)
--- PASS: TestAccS3BucketInventory_encryptWithSSEKMS (65.51s)
--- PASS: TestAccS3Bucket_Basic_namePrefix (71.10s)
--- PASS: TestAccS3Bucket_Security_grantToACL (115.20s)
--- PASS: TestAccS3Bucket_Basic_acceleration (126.33s)
--- PASS: TestAccS3BucketMetric_withEmptyFilter (1.44s)
--- PASS: TestAccS3BucketDataSource_website (54.09s)
--- PASS: TestAccS3BucketInventory_basic (52.80s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_updateBasic (174.91s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_basic (49.70s)
--- PASS: TestAccS3Bucket_Security_policy (180.56s)
--- PASS: TestAccS3BucketMetric_withFilterSingleTag (91.99s)
--- PASS: TestAccS3BucketMetric_withFilterMultipleTags (87.65s)
--- PASS: TestAccS3Bucket_Security_updateGrant (175.79s)
--- PASS: TestAccS3BucketMetric_basic (46.02s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndMultipleTags (84.14s)
--- PASS: TestAccS3BucketDataSource_basic (44.70s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndSingleTag (86.98s)
--- PASS: TestAccS3Bucket_Tags_withNoSystemTags (224.71s)
--- PASS: TestAccS3BucketMetric_withFilterPrefix (75.21s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_jsonEncode (287.54s)
--- PASS: TestAccS3Bucket_Security_updateACL (91.23s)
--- PASS: TestAccS3BucketInventory_encryptWithSSES3 (37.72s)
--- PASS: TestAccS3Bucket_Tags_withSystemTags (252.52s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_Filter (139.54s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutPrefix (326.07s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_tagFilter (320.94s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_andOperator (374.99s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/s3	1030.676s
```

File: .changelog/22613.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+```release-note:note
```
Comment (anGie44) [inline .changelog/22613.txt:1]:
```suggestion
```release-note:breaking-change
```

**Comment by anGie44** [comment]
Output of additional tests changed:
```
--- PASS: TestAccS3BucketRequestPaymentConfiguration_Basic_BucketOwner (32.14s)
--- PASS: TestAccS3BucketRequestPaymentConfiguration_Basic_Requester (32.59s)
--- PASS: TestAccS3BucketRequestPaymentConfiguration_update (80.65s)
```

**Pull request merged**